### PR TITLE
ADCM-1968 Close all tabs in test teardown before container is stopped

### PR DESCRIPTION
**Pull request by skhomuti**

### CODE REVIEW

File: tests/ui_tests/conftest.py [hunk 1]
```diff
@@ -172,12 +171,14 @@ def _write_json_file(f_name, j_data):
 def login_to_adcm_over_api(app_fs, adcm_credentials):
     """Perform login via API call"""
     login_endpoint = f'{app_fs.adcm.url.rstrip("/")}/api/v1/token/'
-    app_fs.driver.get(app_fs.adcm.url)
+    LoginPage(app_fs.driver, app_fs.adcm.url).open().wait_page_is_opened()
```
Comment (Sealwing) [inline tests/ui_tests/conftest.py:174]: You don't need to explicitly wait for the page to be opened, it's implicit in open(). `wait_page_is_opened` exists for cases, when we open page by click on some UI element, for example

Comment (skhomuti) [reply tests/ui_tests/conftest.py]: fixed

File: tests/ui_tests/conftest.py [hunk 2]
```diff
@@ -67,13 +67,6 @@ def app_fs(adcm_fs: ADCM, web_driver: ADCMTest, request):
     Attach ADCM API to ADCMTest object and open new tab in browser for test
     Collect logs on failure and close browser tab after test is done
     """
-    try:
-        web_driver.new_tab()
-    # Recreate session on WebDriverException
-    except WebDriverException:
-        # this exception could be raised in case
-        # when all tabs were closed in process of creating new one
-        web_driver.create_driver()
```
Comment (dgusakov) [inline tests/ui_tests/conftest.py:76]: This route was meant to handle situations of unexpected driver failures. How it will be handled now?

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: tests/ui_tests/conftest.py (hunk 2 above)
Comment (skhomuti) [reply tests/ui_tests/conftest.py:76]: The comment suggests that all tabs were closed, but now it is an impossible case
I have not seen unexpected driver failures. Do you have examples?

**Review by Sealwing — APPROVED** [review]

File: tests/ui_tests/conftest.py (hunk 2 above)
Comment (dgusakov) [reply tests/ui_tests/conftest.py:76]: Nope. But additional insurance is useful)

**Review by Sealwing — APPROVED** [review]

**Review by Sealwing — APPROVED** [review]

**Review by dgusakov — APPROVED** [review]

**Pull request merged**